tests/video/test_video.py

```diff
@@ -298,13 +308,18 @@ def test_reencode_return_ffmpeg_output(
 )
 def test_reencode_failsafe(src, dest, ffmpeg_args, failsafe, hosted_media_links):
     with tempfile.TemporaryDirectory() as temp_dir:
-        temp_dir = pathlib.Path(temp_dir)
-        src_path = temp_dir.joinpath(src)
-        dest_path = temp_dir.joinpath(dest)
-        save_large_file(hosted_media_links[src_path.suffix.replace(".", "")], src_path)
         if not failsafe:
             with pytest.raises(subprocess.CalledProcessError):
-                reencode(src_path, dest_path, ffmpeg_args, failsafe=failsafe)
+                download_media_and_reencode(
+                    temp_dir,
+                    src,
+                    dest,
+                    ffmpeg_args,
+                    hosted_media_links,
+                    failsafe=failsafe,
+                )
```

Comment:
would be nice to also test output without failsafe (from the exception)

Reply:
Done

Reply:
Done